ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml

```
@@ -68,9 +68,12 @@ ref:
   - name: POWERVS_VPC
     default: ""
     documentation: "IBM Cloud VPC Name. This flag is used to reuse an existing VPC resource for cluster's infra"
-  - name: POWERVS_CLOUD_CONNECTION
+  - name: POWERVS_TRANSIT_GATEWAY
     default: ""
-    documentation: "Cloud Connection in given zone.This flag is used to reuse an existing Cloud Connection resource for cluster's infra"
+    documentation: "Transit Gateway in IBM Cloud. This flag is used to reuse an existing Transit Gateway resource for cluster's infra"
+  - name: TRANSIT_GATEWAY_LOCATION
+    default: ""
+    documentation: "Transit Gateway Location to be used for the PowerVS setup."
   - name: CONTROL_PLANE_NODE_SELECTOR
     default: ""
     documentation: "Set nodeSelector for contorl plane pods. If the provided release image is not multi-arch, this is mandatory (e.g., `kubernetes.io/arch=amd64`)."
```

```suggestion
    documentation: "Set nodeSelector for control plane pods. If the provided release image is not multi-arch, this is mandatory (e.g., `kubernetes.io/arch=amd64`)."
```

small spelling change

---

correct it, thanks!